### ISSUE TITLE
About the training of AdaptSegNet(GTA5 -> Cityscapes) and usage of Synscpaes 

### ISSUE THREAD

**Issue by LinfengYuan1997** [wasidennis/AdaptSegNet#98]
Thanks for your  fantastic sharing code!

I have 2 questions about your code:

1. Recently I tried to train your code AdaptSegNet(GTA5 -> Cityscpaes multi ResNet) via PyTorch 1.4.

The only modifications I made are the  argument 'align_corners = True' in nn.Upsample() and small bugs while recording and printing the loss using PyTorch1.x.

And I have trained the model in 120K iterations but only got the highest mIOU on Cityscapes about 40.x% at 40K iteration.

![image](https://user-images.githubusercontent.com/79844318/116188736-7f536880-a75a-11eb-8a44-10389c8f50d3.png)

I am wondering thether it is normal because GAN is unstable, or there are some problems during my training.

And here are some warnings when runnig the training code:
![image](https://user-images.githubusercontent.com/79844318/116188939-c93c4e80-a75a-11eb-84e4-79dddb3ab601.png)
Is there any problem about the optimizer?

2. I plan to use Synscapes to train and validation the UDA of segmentation. I am wondering your setting while using the Synscapes (the train val splits and any hyperparameters tuning while training)

Looking forward to your reply~

**Comment by LinfengYuan1997** [comment]
Maybe all the Synscapes pics will be entered for training because we don't use them while validating on Cityscapes?

Any hyperparameters tuning by the way?
